core/shared/src/main/scala/matryoshka/patterns/Diffable.scala

```diff
@@ -25,7 +25,7 @@ import scalaz._, Scalaz._
 import simulacrum._
 
 @typeclass trait Diffable[F[_]] { self =>
-  private implicit def self0 = self
+  private implicit def selfʹ: Diffable[F] = self
```

Comment:
I forget where exactly, but something in here needs access to a `Diffable` instance, which _can_ be this instance itself, so we make it implicitly available.